src/types/user.ts

```diff
@@ -0,0 +1,60 @@
+import { External } from './external';
+import { Translation } from './translation';
+import { OptionalExclude } from './utils';
+
+// TODO: Type this
```

Comment:
Do you wanna track these TODOs as Jiras or maybe GitHub issues?

Reply:
I think using GitHub issues (or projects) would be better to give visibility to all contributors!